### PR TITLE
Update Turkish translation for the app

### PR DESCRIPTION
**Pull request by safak45x**
new update ! add fix pls

### CODE REVIEW

File: po/tr.po [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 msgstr ""
 "Project-Id-Version: com.github.ryonakano.reco\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-07-23 10:32+0000\n"
-"PO-Revision-Date: 2019-01-16 22:17+0100\n"
-"Last-Translator: Safak Genisol (@safak45x)\n"
-"Language-Team: Turkish\n"
+"PO-Revision-Date: 2020-09-25 09:40+0300\n"
+"Last-Translator: Safak <bjksafak45@gmail.com>\n"
+"Language-Team: Turkish <kde-l10n-tr@kde.org>\n"
```
Comment (ryonakano) [inline po/tr.po:13]:
```suggestion
"Language-Team: Turkish\n"
```

This seems to be the email address set in Lokalize by default

**Review by ryonakano — COMMENTED** [review]
One tiny thing.

**Review by ryonakano — APPROVED** [review]
Looks nice, thanks! :tada:

**Comment by safak45x** [comment]
Thank you for such a nice application.

**Pull request merged**